gent_base.libraries.yml

```diff
@@ -68,3 +68,17 @@ checkbox_filter:
   dependencies:
     - gent_base/global_js
     - gent_base/modal
+
+tabs:
+  version: 1.x
+  js:
+    styleguide/build/styleguide/vendor/allietabs/allieTabs.js: { minified: true }
```

Comment:
Don't we have our own tabs JS? Or am I mistaken?

Reply:
Not really. The tabs are initialized through the DOM with the `data-tabs` attribute 😉